lib/widgets/overview_widget.dart

```diff
@@ -1,9 +1,13 @@
 import 'package:flutter/material.dart';
 import 'package:pokedexbootcamp/api/model/pokemon.dart';
+import 'package:pokedexbootcamp/features/pokemon_details/pokemon_details_connector.dart';
 import 'package:pokedexbootcamp/utils/constants.dart';
 
 class PokemonOverview extends StatelessWidget {
-  const PokemonOverview({Key? key, required this.pokemonName}) : super(key: key);
+  const PokemonOverview({
+    Key? key,
+    required this.pokemonName,
```

Comment:
make it as a habit to place required properties to be at the top of everything 🙂